src/cover/index.ts

```diff
@@ -228,51 +284,50 @@ const opCodes: CoverTableRecipe = Object.fromEntries(
             ]
           : [
               opCode,
-              (lang) =>
-                lang.stmt(
+              (lang) => {
+                const exprs = [false, true].map((forceBuiltin) =>
                   op.unsafe(opCode)(
                     ...getInstantiatedOpCodeArgTypes(opCode).map((t, i) =>
                       lang.expr(
                         opCode === "range_excl" && i === 1
                           ? integerType(10, 10)
                           : t,
+                        forceBuiltin,
                       ),
                     ),
                   ),
-                ),
+                );
+                return lang.stmt(isLiteral(exprs[0]) ? exprs[1] : exprs[0]);
```

Comment:
On main, we were using `(1 + 1)` expressions to test the `+` opcode, but that expression is immediately reduced to `2`, which resulted in some false positives in terms of coverage. So, the failed CI check should be ignored.